pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -780,7 +787,7 @@ func (a *ApplicationRestoreController) restoreVolumes(restore *storkapi.Applicat
 							v1.EventTypeWarning,
 							string(storkapi.ApplicationRestoreStatusFailed),
 							message)
-						_, err = a.updateRestoreCRInVolumeStage(namespacedName, storkapi.ApplicationRestoreStatusFailed, storkapi.ApplicationRestoreStageFinal, message, nil)
+						_, err = a.updateRestoreCRInVolumeStage(namespacedName, storkapi.ApplicationRestoreStatusFailed, storkapi.ApplicationRestoreStageFinal, message, nil, nil)
```

Comment:
why we are passing explicitly "nil" here instead of restore.Spec.NamespaceMapping?

Reply:
First time, when we enter restoreVolume, we will be persisting the namespacemapping to the CR at line 554. In other places, It is not necessary as it is already persisted to the CR, we get it when read the CR content in updateRestoreCRInVolumeStage()